langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -151,6 +205,17 @@ public isolated function getChildren(Element elem) returns xml = @java:Method {
 #
 # This panics if it would result in the element structure
 # becoming cyclic.
+# 
+# ```ballerina
+# xml:Element employees = xml `<employees><employee>David</employee><employee>Peter</employee></employees>`;
+# xml newEmployees = xml `<employee>Alice</employee><employee>Bob</employee>`;
+# employees.setChildren(newEmployees);
+# employees ⇒ <employees><employee>Alice</employee><employee>Bob</employee></employees>
+# 
+# xml:Element x = xml `<student id="1205">John</student>`;
+# x.setChildren("Jane");
+# x ⇒ <student id="1205">Jane</student>
```

Comment:
```suggestion
# xml:Element student = xml `<student id="1205">John</student>`;
# student.setChildren("Jane");
# student ⇒ <student id="1205">Jane</student>
```

For consistency, since the previous variable is named `employees`.